README.md

```diff
@@ -31,12 +31,32 @@ This project aims at bringing the power of QML to the web browser.
 
 ## How to use
 #### Add the library to your web page
+Using one of the methods below, install the qmlweb JavaScript library:
 
-Install with [npm](https://www.npmjs.com/package/qmlweb),
-[Bower](http://bower.io/search/?q=qmlweb), or download directly from the
-GitHub [releases](https://github.com/qmlweb/qmlweb/releases) page.
+- [npm](https://www.npmjs.com/package/qmlweb):
 
-From that package, preload the file `lib/qt.js` in an HTML page.
+  ```
+  npm install qmlweb
+  ```
+
+- [Bower](http://bower.io/search/?q=qmlweb):
+
+  ```
+  bower install qmlweb
+  ```
+
+- GitHub [releases](https://github.com/qmlweb/qmlweb/releases):
+  ```
+  tar -xzvf v0.0.4.tar.gz
+  ```
+
+- Manually using gulp (recommended if you cloned from git):
+  ```
+  npm install
+  gulp build
```

Comment:
@aspidites Sorry, I didn't want to bikeshed, just want everything to be as good as possible =).

Note that the comment is not about the HTML (we already agreed on that, I think), but about replacing `gulp build` with `npm run build`, because `gulp build` will fail for users that don't have `gulp` installed globally, and we don't want that, correct?